frmts/ogcapi/gdalogcapidataset.cpp

```diff
@@ -655,6 +658,44 @@ bool OGCAPIDataset::InitFromFile(GDALOpenInfo *poOpenInfo)
     return InitFromCollection(poOpenInfo, oDoc);
 }
 
+/************************************************************************/
+/*                        ProcessScale()                          */
+/************************************************************************/
+
+bool OGCAPIDataset::ProcessScale(CPLJSONObject &oScaleDenominator,
```

Comment:
Static analyzers will likely complain about bRet being set to FALSE, then TRUE, and the return value being checked and always TRUE.
I would just change the return type to be void

Reply:
It makes sense. I addressed it in this commit: https://github.com/OSGeo/gdal/pull/7656/commits/ca7772ab95e20f32c177be7ed324e3bc1a3aff77